lib/ui/view/draft/draft_viewmodel.dart

```diff
@@ -1,11 +1,189 @@
+import 'dart:convert';
+import 'package:hng/app/app.locator.dart';
+import 'package:hng/app/app.router.dart';
+import 'package:hng/models/draft_data_holder_model.dart';
+import 'package:hng/models/user_post.dart';
+import 'package:hng/services/connectivity_service.dart';
+import 'package:hng/services/local_storage_services.dart';
+import 'package:hng/utilities/enums.dart';
+import 'package:hng/utilities/storage_keys.dart';
 import 'package:hng/constants/app_strings.dart';
 import 'package:stacked/stacked.dart';
-
-import 'drafts.dart';
+import 'package:stacked_services/stacked_services.dart';
+import 'package:hng/app/app.logger.dart';
 
 class DraftViewModel extends BaseViewModel {
-  final _title = Draft;
-  String get title => _title;
+  final _navigationService = locator<NavigationService>();
+  final connectivityService = locator<ConnectivityService>();
+  final _storageService = locator<SharedPreferenceLocalStorage>();
+  final snackBar = locator<SnackbarService>();
+  final log = getLogger("Draft Page View Model");
```

Comment:
There should be no space
It should be DraftPageViewModel not the spaced one

Reply:
lol, done boss

Reply:
done